src/css/tsv.css

```diff
@@ -0,0 +1,59 @@
+.tsv-table {
+    font-feature-settings:"kern";
+    min-width: 100%;
+    border-collapse: collapse;
+    overflow: auto;
+    display: block;
+}
+
+.tsv-table th {
+    color: var(--md-code-fg-color);
+    font-family:var(--md-code-font-family);
+    font-weight: 800;
+    background-color: var(--md-code-bg-color);
+    border-top: 1px solid var(--md-default-fg-color--light);
+    font-size: 0.8em;
+    padding: 0 10px 0 10px;
+}
+
+.tsv-table.index tr th:first-child {
+    background-color: var(--md-text-bg-color);
+    border-top: 0;
+    user-select: none;
+}
+
+.tsv-table tr:last-child td {
+    border-bottom: 1px solid var(--md-default-fg-color--light);
+}
+
+.tsv-table tr:first-child td {
+    border-top: 1px solid var(--md-default-fg-color--light);
+    user-select: none;
```

Comment:
Nevermind. This did the wrong thing. I thought it wasn't working because I had the old CSS cached.

```suggestion
```